src/test/compile-fail/impl-wrong-item-for-trait.rs

```diff
@@ -12,7 +12,9 @@
 
 trait Foo {
     fn bar(&self);
-    const MY_CONST: u32;
+    //~^ NOTE original trait requirement
+    //~| NOTE original trait requirement
```

Comment:
I needed to put that note twice because there are 2 different tests that display that note, is that normal or a bug?